localstack/services/awslambda/lambda_executors.py

```diff
@@ -141,7 +144,11 @@ def __init__(
         self.invoked_function_arn = lambda_function.arn()
         if qualifier:
             self.invoked_function_arn += ":" + qualifier
-        self.cognito_identity = context.get("identity")
+        self.cognito_identity = context.get("identity") and {
```

Comment:
nit: We could use `select_attributes(..)` from the `collections` utils here. 👍

Reply:
done!